src/backend/gporca/data/dxl/minidump/JoinCombinedHashSpecNullsNotColocated.mdp

```diff
@@ -0,0 +1,1379 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<dxl:DXLMessage xmlns:dxl="http://greenplum.com/dxl/2010/12/">
+  <dxl:Comment><![CDATA[
+  Test that matching hashed distribution for the combined distribution of HOJ
+  is calculated correctly. In case below the required distribution of the
+  Hash Anti Join's outer child should be calculated based not on the nullable
+  part of HOJ (inner child, t2_1.j), but on the part where nulls are co-located
+  (t1_1.i). Otherwise, there could be an extra Redistribute Motion on the
+  outer child, what lead to the inconsistent join result.
+
+  DROP TABLE IF EXISTS t1, t2;
+  CREATE TABLE t1 (i int) DISTRIBUTED RANDOMLY;
+  CREATE TABLE t2 (j int) DISTRIBUTED RANDOMLY;
+
+  INSERT INTO t1 SELECT i FROM generate_series (500, 1500) i;
```

Comment:
Could you please rebuild the test with less amount of values? It would be more lightweight.
```
insert into t1 select i from generate_series (1, 20) i;
insert into t2 select i from generate_series (1, 10) i;
```